docs/docsite/rst/plugins/lookup.rst

```diff
@@ -56,6 +58,19 @@ You can combine lookups with :ref:`filters <playbooks_filters>`, :ref:`tests <pl
         - "{{ lookup('sequence', 'end=42 start=2 step=2')|map('log', 4)|list) }}"
         - ['a', 'c', 'd', 'c']
 
+Understanding lookup plugin paths
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Lookups are executed with a working directory relative to the role or play (as opposed to local tasks, which are executed relative to the executed script). When searching for files and other resources, however, some lookups search additional paths.
```

Comment:
we do not change working directory, this should read 'relative paths are calculated ...'also its not to play, bu to the file where the task is defined and/or role(s) it belongs to, link to the search paths docs as that is what this follows.

Reply:
with that reference you should be able to skip most of what is below